lib/Doctrine/Deprecations/Deprecation.php

```diff
@@ -56,6 +56,9 @@ class Deprecation
     private static $ignoredPackages = [];
 
     /** @var array<string,int> */
+    private static $triggeredLinkCounts = [];
```

Comment:
I suggest naming this property `$triggeredDeprecations`, like its getter

Reply:
Even beter, changed.